lib/ansible/modules/windows/win_copy.py

```diff
@@ -29,21 +29,44 @@
 version_added: "1.9.2"
 short_description: Copies files to remote locations on windows hosts.
 description:
-     - The C(win_copy) module copies a file on the local box to remote windows locations.
+    - The C(win_copy) module copies a file on the local box to remote windows
+      locations.
 options:
-  src:
-    description:
-      - Local path to a file to copy to the remote server; can be absolute or relative.
-        If path is a directory, it is copied recursively. In this case, if path ends
-        with "/", only inside contents of that directory are copied to destination.
-        Otherwise, if it does not end with "/", the directory itself with all contents
-        is copied. This behavior is similar to Rsync.
-    required: true
-  dest:
-    description:
-      - Remote absolute path where the file should be copied to. If src is a directory,
-        this must be a directory too. Use \\ for path separators.
-    required: true
+    content:
+        description:
+            - When used instead of C(src), sets the contents of a file directly
+              to the specified value. This is for simple values, for anything
+              complex or with formatting please switch to the template module.
+        required: False
+        version_added: "2.3"
+    dest:
+        description:
+            - Remote absolute path where the file should be copied to. If src
+              is a directory, this must be a directory too. Use \ for path
+              separators.
+        required: True
+    remote_src:
+        description:
+            - If False, it will search for src at originating/master machine,
+              if True it will go to the remote/target machine for the src.
+        required: False
+        default: False
```

Comment:
This is a Boolean flag, if set will copy the files paths of the remote hosts path and not from ansible to windows like normal. Is that ok with you?